src/test/java/com/karankumar/bookproject/backend/service/UserServiceTest.java

```diff
@@ -0,0 +1,144 @@
+package com.karankumar.bookproject.backend.service;
+
+import com.karankumar.bookproject.annotations.IntegrationTest;
+import com.karankumar.bookproject.backend.entity.account.Role;
+import com.karankumar.bookproject.backend.entity.account.User;
+import com.karankumar.bookproject.backend.repository.RoleRepository;
+import com.karankumar.bookproject.backend.repository.UserRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.AuthenticationServiceException;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.validation.ConstraintViolationException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
As we are not using wildcard imports, could you change the occurrences of these static imports to use the qualified class name, instead?

Reply:
Do you consider the static imports wildcard imports?
As far as I know wildcard imports are `import a.b.c.*`.

In my opinion, static imports greatly improve the readability in tests, though that might be personal preference.

Reply:
Personally, I consider any use of the wildcard (`*`) to be a wildcard import, but we did explicitly qualify this in the style guide (which Google also did with their Java style guide).

I can see where you're coming from. I would be happy to debate this over Gitter, as I'm always happy for us to change any of the style guide rules.

Reply:
OK, my bad. We're agreeing on wildcard imports are, but I made a mistake by commenting on a static, non-wildcard import. Ignore me :)